exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -125,8 +133,6 @@ def _create_database_container(self, db_ip_address: str, db_private_network: str
                     ports=ports,
                     runtime=self.docker_runtime
                 )
-            if self.db_os_access == DbOsAccess.SSH:
-                self._enable_ssh_access(db_container)
```

Comment:
we still need this, in case of a c4 based docker images

Reply:
will be re-added in next push